tests/hash/test_parse_checksum.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+import tempfile
+import unittest
+import urllib
+
+import mozfile
+
+import mozdownload
+from mozdownload.scraper import BASE_URL
+from mozdownload.utils import urljoin
+
+
+tests_release_scraper = [
+    # -p win32 -v latest
+    {'args': {'application': 'firefox',
+              'platform': 'win32',
+              'version': '52.0.2'},
+     'url': 'https://archive.mozilla.org/pub/firefox/releases/52.0.2/SHA512SUMS',
```

Comment:
Please do not use `archive.mozilla.org` as URL for the checksums file. It should be all added as files to the appropriate sub folders under `tests/data`.